docker/env/gafl_webapp-telesales.secrets.env

```diff
@@ -0,0 +1,4 @@
+####################################################################################
```

Comment:
Really picky but could we use _ in the filename for consistency with everything else.

Reply:
done